src/block/Sugarcane.php

```diff
@@ -64,7 +73,7 @@ private function grow() : bool{
 				}
 				$this->position->getWorld()->setBlock($b->position, $ev->getNewState());
 				$grew = true;
-			}else{
+			}elseif(!$b instanceof Sugarcane){
```

Comment:
```suggestion
			}elseif(!$b->isSameType($this)){
```

Reply:
Just out of curiosity, what is the difference between the two?